shared/fixes/bash/ensure_redhat_gpgkey_installed.sh

```diff
@@ -14,6 +14,8 @@ then
   # (to ensure there won't be e.g. CRC error).
   IFS=$'\n' GPG_OUT=($(gpg --with-fingerprint "${REDHAT_RELEASE_KEY}" | grep 'Key fingerprint ='))
```

Comment:
Wouldn't it be more elegant to set the IFS inside the subshell? That way there is no need to unset it.

Reply:
@redhatrises I had already grepped the code base for unmatched instances of IFS= that didn't have a corresponding unset IFS before submitting this.  The one exception is shared/bash_remediation_functions/perform_audit_rules_privileged_commands_remediation.sh but I don't really have time to make that change and confirm that I didn't accidentally break anything.

@mpreisler if someone else wants to take the effort to rewrite these statements they are welcome to.  I'm limiting the scope of fixes to things that affect remediations I am running on my systems due to limited time.  I was keeping this fix similar to how the IFS / unset IFS behavior is already implemented in other scripts to avoid additional troubleshooting time.

Reply:
ok, makes sense to keep all of these patterns the same then